src/test/java/uk/gov/mca/beacons/service/mappers/BeaconMapperUnitTest.java

```diff
@@ -95,9 +88,18 @@ void shouldThrowAnExceptionIfItCannotParseAValidDate() {
 
     assertThrows(
       DateTimeException.class,
-      () -> {
-        beaconMapper.fromDTO(beaconDTO);
-      }
+      () -> beaconMapper.fromDTO(beaconDTO)
     );
   }
+
+  @Test
+  void shouldAccuratelyStripTimeInformationFromDateFields() {
+    beaconDTO.addAttribute("batteryExpiryDate", "2022-02-01T00:00");
+    beaconDTO.addAttribute("lastServicedDate", "2019-02-01T00:00");
+
+    var beacon = beaconMapper.fromDTO(beaconDTO);
+
+    assertThat(beacon.getBatteryExpiryDate(), is(LocalDate.of(2022, 2, 1)));
+    assertThat(beacon.getLastServicedDate(), is(LocalDate.of(2019, 2, 1)));
+  }
```

Comment:
Defensive for cases where a client submits a DateTime.  Took the decision to be timezone agnostic and just strip off the time/timezone information from the end of the string.